addons/point_of_sale/static/src/app/navbar/closing_popup/closing_popup.js

```diff
@@ -15,7 +15,19 @@ import { deduceUrl } from "@point_of_sale/utils";
 export class CancelConfirmationDialog extends ConfirmationDialog {
     static template = "point_of_sale.CancelConfirmationDialog";
 }
-
+export class DeleteConfirmationDialog extends AlertDialog {
+    static template = "point_of_sale.DeleteConfirmationDialog";
+    setup() {
+        this.pos = usePos();
+    }
+    deleteOrder() {
+        const orders = this.pos.models["pos.order"].getAll().filter((x) => x.state === "draft");
+        orders.forEach((order) => {
+            this.pos.removeOrder(order, true);
+        });
+        this.props.close();
+    }
+}
```

Comment:
There is no need to create a new dialog for this. You can simply use the confirmation dialog